versions.props

```diff
@@ -99,7 +100,13 @@ org.rocksdb:rocksdbjni = 6.4.6
 org.slf4j:* = 1.7.25
 org.yaml:snakeyaml = 1.23
 org.freemarker:freemarker = 2.3.29
+com.palantir.refreshable:refreshable = 0.1.0
 
 com.google.errorprone:error_prone_refaster = 2.3.3
 com.palantir.baseline:baseline-refaster-testing = 2.17.0
 com.google.truth:truth = 1.0
+
+
+# conflict resolution
+org.apache.httpcomponents:httpclient = 4.5.12
```

Comment:
Where is the conflict coming from? Is there a better way to resolve it? it would be good to instruct gradle to just take the dialogue's version.

Reply:
Unless this is coming from some Oracle related code, in which case we'll need to check those versions make sense.

Reply:
These come from cassandra-thrift (including latest version). I'm checking with @leonz if this is safe and/or what we can do here.

Reply:
Turns out this is tricky to bump. But given this combination has _already_ been running in a fair few places and the [discrepancy](https://archive.apache.org/dist/httpcomponents/httpclient/RELEASE_NOTES-4.5.x.txt) is small I think we can take on this risk.